framework/src/modules/http_api/defaults/config.js

```diff
@@ -26,10 +30,12 @@ const DefaultConfig = {
 				public: {
 					type: 'boolean',
 					default: false,
+					env: 'LISK_API_PUBLIC',
 				},
 				whiteList: {
 					type: 'array',
 					default: ['127.0.0.1'],
+					env: ['LISK_API_WHITELIST', 'stringToIpPortSet'],
```

Comment:
i think env and formatter should be separate property.
ie)
env: 'LISK_API_WHITELIST',
parser: 'stringToIPPortSet',
Also default doesn't have port setup

Reply:
Passing parser as separate key to parent schema applies it to json config as well. If you want we can make `env` key it be an object and then specify parser.